x-pack/plugins/security_solution/public/flyout/entity_details/host_right/index.tsx

```diff
@@ -120,6 +123,26 @@ export const HostPanel = ({
 
   const hasVulnerabilitiesFindings = sum(Object.values(vulnerabilitiesData?.count || {})) > 0;
 
+  const { signalIndexName } = useSignalIndex();
+
+  const entityFilter = useMemo(() => ({ field: 'host.name', value: hostName }), [hostName]);
+
+  const { items: alertsData } = useAlertsByStatus({
+    entityFilter,
+    signalIndexName,
+    queryId: `${DETECTION_RESPONSE_ALERTS_BY_STATUS_ID}HOST_NAME_RIGHT`,
+    to,
+    from,
+  });
+
+  const alertsOpenCount = alertsData?.open?.total || 0;
+
+  const alertsAcknowledgedCount = alertsData?.acknowledged?.total || 0;
+
+  const alertsCount = alertsOpenCount + alertsAcknowledgedCount;
+
+  const hasNonClosedAlerts = alertsCount;
```

Comment:
nit: When a variable name starts with "has", it is expected to be a boolean.
```suggestion
  const hasNonClosedAlerts = (alertsData?.acknowledged?.total || 0 + alertsData?.open?.total || 0) > 0;
```